api-sdk/src/main/java/com/smartling/api/sdk/auth/RefreshCommand.java

```diff
@@ -0,0 +1,43 @@
+package com.smartling.api.sdk.auth;
+
+import java.util.Objects;
+
+public class RefreshCommand
+{
+    private final String refreshToken;
+
+    public RefreshCommand(String refreshToken)
+    {
+        this.refreshToken = refreshToken;
+    }
+
+    public String getRefreshToken()
+    {
+        return refreshToken;
+    }
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
+        RefreshCommand that = (RefreshCommand) o;
+        return refreshToken.equals(that.refreshToken);
```

Comment:
what if accidentally refreshToken was filled as null? maybe use Objects.equals(refreshToken, that)

Reply:
It mustn't be null
because this is a **refresh** method
It doesn't matter what we throw - NPE or another custom exception

Reply:
If you mean it's checked in context above - then ok. But it's a public class and equals potentially can throw npe.